lib/logstash/outputs/loggly.rb

```diff
@@ -100,8 +100,18 @@ def receive(event)
     # we should ship logs with the default tag value.
     tag = 'logstash' if /^%{\w+}/.match(tag)
 
+    # Duplicate timestamp to a field Loggly will recognize
+    remove_timestamp = false
```

Comment:
to make this simpler and less error prone (imagine if there's an error processing the event, it may carry on without the remove operation happening) we can go back to using:

```ruby
event_hash = event.to_hash
event_hash["timestamp"] = event.timestamp unless event_hash.include?("timestamp")
send_data(event_hash.to_json) # pseudo code
```

Reply:
it's somewhat slower but I hardly see it has being the bottleneck in this plugin